code/_compile_options.dm

```diff
@@ -70,6 +70,12 @@
 #error You need version 514.1556 or higher
 #endif
 
+#if (DM_VERSION == 514 && DM_BUILD > 1571)
```

Comment:
i have 1575 and ive not ran into any issues

Reply:
i must of fucked up updating to that version when testing, i'll adjust it